signing_service/src/signing_service.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+import logging.config
+import os
+import socket
+
+
+logger = logging.getLogger()
+crash_logger = logging.getLogger('crash')
+
+
+def main(concent_cluster_address):
+    address, port = concent_cluster_address.split(':')
+    port = int(port)
+
+    socket_to_concent = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    socket_to_concent.connect((address, port))
+
+    socket_to_concent.send(b'GET / HTTP/1.0\r\n\r\n')
```

Comment:
Maybe let's not pretend that it's HTTP. Might be confusing during debugging. Please just send some random sentence.

Reply:
Concent, as a HTTP server, is not responding to a random sentence.

Reply:
Oh, so you're abusing a HTTP sever to test it instead of writing a custom server :) Clever. It's fine then.

Reply:
I have written custom one for #634 ;) And issue actually mentions that it should communicate with the Concent.

Reply:
With Concent cluster. And Middleman is running as a service on the Concent cluster.

Reply:
From the perspective of the Signing Service it does not really matter what it talks to. Only that it's using the right protocol. And that protocol is not HTTP :)

Reply:
Right, but it is not available yet. Anyway, I have custom server for #634 . I also have an idea to talk about on retro to create a placeholder for reusable scripts, commands and tools like this that are not in our repository.

Reply:
If it's really reusable then just create a directory called `tools/` and put it there. I don't think this would be a controversial thing go do unless you're adding something that is huge or is really a one-off.

Reply:
I actually mean something more like drafts, not necessary matching our requirements like coding style etc. Also, commands for testing (or set of commands) like longer `curl` requests etc. Generally everything that one developer has on his local machine and other developer might find useful. But lets discuss this on retro.